catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/converter/TestConvertUtil.java

```diff
@@ -278,12 +278,20 @@ public void testFormIcebergType() {
     Assertions.assertTrue(
```

Comment:
could you add a integrate test in `CatalogIcebergIT`?

Reply:
updated.